gateway/src/config.rs

```diff
@@ -105,7 +105,7 @@ impl TracingTelemetry {
                 let used_endpoint = match endpoint {
                     Endpoint::Kube { kube_node_ip_env } => format!(
                         "http://{}:8126",
-                        std::env::var(kube_node_ip_env).expect(&format!(
+                        std::env::var(kube_node_ip_env).unwrap_or_else(|_| panic!(
```

Comment:
If we really do want to panic, maybe `#[allow(clippy::whatever_it_is)]` is better in this case?

Reply:
I'm fine with both as long as the warning disappears from my editor ;). I'll let the gateway maintainers decide which one they prefer.

Reply:
The warning is https://rust-lang.github.io/rust-clippy/master/index.html#/expect_fun_call

The point isn't to prevent the panic, it's to prevent the unnecessary call to format in the happy case.